products/sle/main.pm

```diff
@@ -464,25 +464,30 @@ sub load_slenkins_tests {
     return 0;
 }
 
-sub load_cluster_boot {
-    if (check_var('ARCH', 'aarch64')) {
-        # We need to boot *before* waiting for support server
-        # Because of TianoCore UEFI boot method
-        loadtest "boot/boot_to_desktop";
-        # Wait for support server to complete its initialization
-        loadtest "support_server/wait_support_server";
-        # TODO: poo#32110 add reboot or ensure that network is up
+sub boot_hpc {
```

Comment:
Please don't rename it. This function can be shared by other tests, not only by HPC.
I have in plan to use it for network clusters in the future.